core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -813,10 +813,10 @@ public SystemSessionProperties(
                         "Target size in bytes of all task inputs for a single fault tolerant task",
                         queryManagerConfig.getFaultTolerantExecutionTargetTaskInputSize(),
                         false),
-                integerProperty(
-                        FAULT_TOLERANT_EXECUTION_TARGET_TASK_SPLIT_COUNT,
-                        "Target number of splits for a single fault tolerant task (split weight aware)",
-                        queryManagerConfig.getFaultTolerantExecutionTargetTaskSplitCount(),
+                dataSizeProperty(
+                        FAULT_TOLERANT_EXECUTION_STANDARD_SPLIT_SIZE,
+                        "Standard split size for a single fault tolerant task (split weight aware)",
```

Comment:
nit: define description as constant in Config class and use it here

Reply:
You mean `QueryManagerConfig.java`?